install.sh

```diff
@@ -162,12 +162,21 @@ else
 fi
 
 # Choose the right GHC download
+case `uname -m` in
+  i386)   GHC_CPU=i386;;
+  i686)   GHC_CPU=i386;;
+  x86_64) GHC_CPU=x86_64;;
+  amd7)   GHC_CPU=amd7;;
+  *) >&2 echo "Unrecognized machine: ${MACHINE}"; exit 1;;
+esac
+
+## then
```

Comment:
Stray comment?  I'm not sure what it meant.

Reply:
the ```## then```? Thought it was for originally,  go through cases ```##then``` choose appropriate download

Reply:
Oh, I just re-pasted the formatting so that it matched what I had previously. It was previously there..didn't actually change anything.

Reply:
Yes, I meant `## then`.  It is not there before this change, so I suppose it just snuck in by accident.  Can you remove it, please?